src/store/components/AdminItemPage/index.tsx

```diff
@@ -22,79 +23,115 @@ interface AdminItemPageProps {
+  lifetimeLimit: Yup.number().min(1, 'Too Low').max(10, 'Too High').required('Required'),
+  categoryName: Yup.string().when('hasVariantsEnabled', {
+    is: (hasVariantsEnabled) => hasVariantsEnabled,
+    then: Yup.string().min(2, 'Too Short').max(50, 'Too Long').required('Required'),
+  }),
+  options: Yup.array().when('hasVariantsEnabled', {
+    is: (hasVariantsEnabled) => hasVariantsEnabled,
+    then: Yup.array().of(
+      Yup.object().shape({
+        uuid: Yup.string(),
+        value: Yup.string().min(2, 'Too Short').max(50, 'Too Long').required('Required'),
+        price: Yup.number().min(1, 'Too Low').max(1000000, 'Too High').required('Required'),
+        quantity: Yup.number().min(1, 'Too Low').max(1000000, 'Too High').required('Required'),
+        discountPercentage: Yup.number().min(0, 'Too Low').max(100, 'Too High').required('Required'),
+      }),
+    ),
+  }),
+  quantity: Yup.number().when('hasVariantsEnabled', {
+    is: (hasVariantsEnabled) => !hasVariantsEnabled,
+    then: Yup.number().min(1, 'Too Low').max(1000000, 'Too High').required('Required'),
+  }),
+  price: Yup.number().when('hasVariantsEnabled', {
+    is: (hasVariantsEnabled) => !hasVariantsEnabled,
+    then: Yup.number().min(1, 'Too Low').max(1000000, 'Too High').required('Required'),
+  }),
+  discountPercentage: Yup.number().when('hasVariantsEnabled', {
+    is: (hasVariantsEnabled) => !hasVariantsEnabled,
+    then: Yup.number().min(0, 'Too Low').max(100, 'Too High').required('Required'),
+  }),
+});
+
 const AdminItemPage: React.FC<AdminItemPageProps> = (props) => {
   const { item, collections } = props;
 
   const creatingItem = !item;
 
+  const initialValues: AdminItemPageForm = {
+    collection: item?.collection.uuid ?? '',
+    itemName: item?.itemName ?? '',
+    description: item?.description ?? '',
+    existingPicture: item?.picture,
+    newPicture: undefined,
+    hidden: false, // TODO: Backend doesn't currently return this property, so we don't know its state.
+    monthlyLimit: item?.monthlyLimit.toString() ?? '',
+    lifetimeLimit: item?.lifetimeLimit.toString() ?? '',
+    hasVariantsEnabled: item?.hasVariantsEnabled ?? false,
+    // Variants Enabled
+    categoryName: (item?.hasVariantsEnabled && item?.options[0].metadata?.type) || '',
+    options: item?.hasVariantsEnabled
+      ? item?.options.map((option) => ({
+          uuid: option.uuid,
+          value: option.metadata?.value ?? '',
+          price: option.price.toString(),
+          quantity: option.quantity.toString(),
+          discountPercentage: option.discountPercentage.toString(),
+        }))
+      : [],
+    // Variants Disabled
+    quantity: (!item?.hasVariantsEnabled && item?.options[0].quantity?.toString()) || '',
+    price: (!item?.hasVariantsEnabled && item?.options[0].price.toString()) || '',
+    discountPercentage: (!item?.hasVariantsEnabled && item?.options[0].discountPercentage.toString()) || '',
+  };
+
```

Comment:
`initialValues` was moved outside of the `<Formik>` tag so typing could be applied.